mx.py

```diff
@@ -4772,9 +4777,8 @@ def expand(value, context):
             'developer',
             'url',
             'licenses',
-            'licences',
             'defaultLicense',
-            'defaultLicence',
```

Comment:
I am sorry guys. I thought the keys are duplicated. I haven't noticed the difference in "s" vs. "c". I update my diff.